oxidation/libparsec/crates/crypto/tests/sequester.rs

```diff
@@ -305,3 +305,18 @@ fn test_encrypt_compat() {
 
     assert_eq!(priv_key.decrypt(&encrypted).unwrap(), b"Hello world");
 }
+
+#[test]
+fn verify_with_different_salt_len() {
+    let (signing_key94, verify_key94) =
+        SequesterSigningKeyDer::generate_pair_with_salt_len(SequesterKeySize::_1024Bits, 94);
```

Comment:
use already generated data instead of generating it on the fly: this prevent the same library from doing data generation and data validation while we want to test cross platform compatibility !

Reply:
Done